decidim-core/app/mailers/concerns/decidim/multitenant_asset_host.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A module to be included in the mailers that allows
+  # to define the assets host based on an organization host
+  module MultitenantAssetHost
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :set_asset_host
+
+      # set_asset_host allows to redefine the host asset if not specified in application configuration
+      # We use a lambda because this code is executed when the application is started
+      # and does not allow us to modify it afterwards.
+      # Leave action_mailer asset_host emtpy to use it.
```

Comment:
```suggestion
      # Leave action_mailer asset_host empty to use it.
```
Just a little typo here :)